scripts/test

```diff
@@ -9,6 +9,12 @@ set -x
 
 export CATTLE_TEST_AGENT_IMAGE PHANTOMJS_BIN
 
+if [ -z "${CATTLE_TEST_PARALLEL_THREADS}" ]; then
```

Comment:
Will make this change:
: ${CATTLE_TEST_PARALLEL_THREADS:=1}
export CATTLE_TEST_AGENT_IMAGE PHANTOMJS_BIN CATTLE_TEST_PARALLEL_THREADS